c/src/ml-api-inference-single.c

```diff
@@ -1078,10 +1075,15 @@ ml_single_open_custom (ml_single_h * single, ml_single_preset * info)
     fw_name = _ml_get_nnfw_subplugin_name (nnfw);       /* retry for "auto" */
   }
   hw_name = _ml_nnfw_to_str_prop (hw);
+
   g_object_set (filter_obj, "framework", fw_name, "accelerator", hw_name,
-      "model", converted_models, NULL);
+      "model", converted_models, "invoke-dynamic", info->invoke_dynamic,
+      "invoke-async", info->invoke_async, NULL);
```

Comment:
In the current tizen nnstreamer `invoke-async` prop is not yet introduced thus emit errors:
`g_object_set_is_valid_property: object class 'GTensorFilterSingle' has no property named 'invoke-async'`.
But it doesn't seem to be harmful.

Reply:
Okay, I will remove this. Thank you.